blocks/gnav/gnav.css

```diff
@@ -1,389 +1,538 @@
+  border: 1px solid #E1E1E1;
+  border-radius: 0 0 4px 4px;
 }
 
-header .gnav .gnav-search-highlight {
-  background-color: yellow;
+.gnav-profile.is-Open .gnav-profile-menu {
+  display: unset;
 }
 
-@media (max-width: 1199px) {
-  
-  header .gnav[aria-expanded=true] .gnav-hamburger::before {
-    content: "\2715";
-  }
-  
-  header .gnav[aria-expanded=true] {
-    grid-template-areas: 
-    'hamburger logo    signin'
-    'search    search  search'
-    'section   section section';
-    grid-template-rows: 64px;
-    height: unset;
-    min-height: 200px;
-  }
-  
-  header .gnav[aria-expanded=true] .gnav-search,
-  header .gnav[aria-expanded=true] .gnav-section {
-    display: block;
-    margin: 0;
-    width: 100vw;
-    box-sizing: border-box;
-    margin-left: -23px;
-  }
-  
-  header .gnav[aria-expanded=true] .gnav-section > span {
-    padding: 16px 20px 16px 35px;
-    border-bottom: 1px solid #f4f4f4;
-  }
-  
-  header .gnav[aria-expanded=true] .gnav-search {
-    background-color: white;
-    padding: 20px;  
-  }
-  
-  header .gnav[aria-expanded=true] .gnav-section {
-    background-color: #fbfbfb;
-  }  
+.gnav-profile-header {
+  display: flex;
+  padding: 20px;
+  align-items: start;
 }
 
-@media (min-width: 1200px) {
+.gnav-profile-header .gnav-profile-img {
+  width: 64px;
+  height: 64px;
+  border-radius: 4px;
+}
 
-  header .gnav .gnav-hamburger {
-    display: none;
-  }
+.gnav-profile-details {
+  margin-left: 16px;
+  align-self: stretch;
+  display: flex;
+  flex-wrap: wrap;
+}
 
-  header .gnav .gnav-logo {
-    grid-area: logo;
-    justify-self: start;
-  }
-  
+.gnav-profile-name {
+  font-size: 18px;
+  line-height: 1;
+  margin: 0;
+  font-weight: 700;
+  flex: 0 0 100%;
+}
 
-  header .gnav .gnav-logo,
-  header .gnav .gnav-section {
-    display: block;
-  }  
-  
-  header .gnav {
-    color: #2c2c2c;
-    max-width: 1440px;
-    min-width: 1200px;
-    margin: auto;
-    grid-template-areas: 'logo section search signin';
-    grid-template-columns: 200px 1fr 20px 50px;
-  }
+.gnav-profile-email {
+  color: #707070;
+  font-size: 14px;
+  line-height: 1.25;
+  margin: 0;
+  flex: 0 0 100%;
+}
 
-  header .gnav .gnav-section {
-    padding: 0 8.5px 1px 8.5px;
-  }
+.gnav-profile-account {
+  margin: 0;
+  text-decoration: underline;
+  font-size: 14px;
+  align-self: flex-end;
+  flex: 0 0 100%;
+}
 
-  header .gnav .gnav-right {
-    padding: 0 10px;
-  }
+.gnav-profile-actions {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  font-size: 14px;
+}
 
-  header .gnav .gnav-section > span {
-    display: inline-flex;
-    height: 64px;
-    align-items: center;
-  }
+.gnav-profile-actions a {
+  padding: 14px 20px;
+  display: block;
+  border-top: 1px solid #E1E1E1;
+  box-sizing: border-box;
+  color: #4b4b4b;
+}
 
-  header .gnav .gnav-section > span.gnav-selected {
-    padding-top: 2px;
-    border-bottom: 2px solid #2c2c2c;
+.gnav-profile-actions a:hover {
+  background-color: #FAFAFA;
```

Comment:
the issue is barely noticeable, but when you hover on "sign out" the rounded corners are not fully visible
<img width="372" alt="Screen Shot 2021-10-06 at 16 04 43" src="https://user-images.githubusercontent.com/408175/136207874-50875e8c-8455-45d9-9dac-6daa9a00383a.png">